package.json

```diff
@@ -22,28 +22,26 @@
     "watch": "webpack --progress --colors --watch --watch-poll"
   },
   "devDependencies": {
-    "babel-core": "^6.23.1",
-    "babel-eslint": "^7.1.1",
-    "babel-loader": "^6.3.2",
-    "babel-polyfill": "^6.22.0",
-    "babel-preset-es2015": "^6.22.0",
-    "base64-loader": "^1.0.0",
```

Comment:
Previously we set these to `^` because babel is particularly spammy re: Greenkeeper, and because we trust babel to stick to the true meaning of semver, so won't release breaking changes here.